docs/desktop-wpf/migration/differences-from-net-framework.md

```diff
@@ -22,8 +22,8 @@ There are a few features of .NET Framework that .NET Core doesn't support. For m
 
 If your .NET Framework app lists its NuGet dependencies in a *packages.config* file, migrate to the [`<PackageReference>`](/nuget/consume-packages/package-references-in-project-files) format.
 
-01. In Visual Studio, find the *Solution Explorer* pane.
-01. With your WPF app, right-click **packages.config** > **Migrate packages.config to PackageReference**.
+01. In Visual Studio, find the **Solution Explorer** pane.
+01. Within your WPF project, right-click *packages.config* > **Migrate packages.config to PackageReference...**.
```

Comment:
Our guidelines specifically say we don't show the ... (I'll get the exact text)

```suggestion
01. Within your WPF project, right-click *packages.config* > **Migrate packages.config to PackageReference**.
```

Reply:
Here it is: https://docs.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions 

If an option label ends with a colon or an ellipsis, don't include that end punctuation in instructions.

Reply:
@mairaw, Thanks! Didn't know that.